packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -448,6 +458,14 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions): Pro
 
   let usersWithCredentials = hosts.map(({ isFixed, user }) => ({ isFixed, ...user }));
 
+  if (eventType.schedulingType && eventType.team) {
```

Comment:
We need to check here if it is a team even-type because dynamic event types are Collective event types but don't have team members. This resulted in empty availability for dynamic event types and caused `dynamic-bookings-pages.e2e.ts` to fail